api/admin/controller.py

```diff
@@ -73,6 +76,7 @@ def setup_admin_controllers(manager):
     manager.admin_work_controller = WorkController(manager)
     manager.admin_sign_in_controller = SignInController(manager)
     manager.admin_feed_controller = FeedController(manager)
+    manager.admin_dashboard_controller = DashboardController(manager)
```

Comment:
The circulation event methods were in the FeedController even though they don't return feeds. I made a new controller for those and the new stats endpoint.